### PR TITLE
called multiple time - so check if already defined

### PR DESCRIPTION
**Pull request by migoi**
## Description

called multiple time, a second define will fail for same index

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code is written according to the guidelines found [here] (https://yetiforce.com/en/developer-documentation/standards/167-how-to-create-php-files.html).
- [x] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: modules/Vtiger/actions/Save.php [hunk 1]
```diff
@@ -78,8 +78,12 @@ public function process(Vtiger_Request $request)
 
 	public function postProcess(Vtiger_Request $request)
 	{
-		define('_PROCESS_TYPE', 'View');
-		define('_PROCESS_NAME', 'Detail');
+		if (!defined('_PROCESS_TYPE')) {
+			define('_PROCESS_TYPE', 'View');  // fixme hardcoded and via define?
```
Comment (mariuszkrzaczkowski) [inline modules/Vtiger/actions/Save.php]: It must override values

File: modules/Vtiger/actions/Save.php [hunk 2]
```diff
@@ -78,8 +78,12 @@ public function process(Vtiger_Request $request)
...
+		}
+		if (!defined('_PROCESS_NAME')) {
+			define('_PROCESS_NAME', 'Detail'); // fixme hardcoded and via define?
```
Comment (mariuszkrzaczkowski) [inline modules/Vtiger/actions/Save.php]: It must override values

Comment (migoi) [inline modules/Vtiger/actions/Save.php]: you cannot override a define (constant)

File: modules/Vtiger/actions/Save.php (hunk 1 above)
Comment (migoi) [inline modules/Vtiger/actions/Save.php]: you cannot override a define (constant)

**Pull request closed without merging**